src/Qowaiv/SvoBehavior.cs

```diff
@@ -0,0 +1,235 @@
+    /// <param name="str">
+    /// The input string to validate.
+    /// </param>
+    /// <param name="formatProvider">
+    /// The optional format provider.
+    /// </param>
+    /// <param name="validated">
+    /// The validated (and potentially transformed) result.
+    /// </param>
+    [Pure]
+    public virtual bool IsValid(string str, IFormatProvider? formatProvider, out string validated)
+    {
+        validated = str;
+        return true;
+    }
+
+    /// <inheritdoc />
+    [Pure]
+    public virtual int Compare(string? x, string? y) => Comparer<string>.Default.Compare(x!, y!);
+
+    /// <summary>Gets the length of string representation of Single Value Object.</summary>
+    [Pure]
+    public virtual int Length(string str) => str?.Length ?? 0;
+
+    /// <summary>Normalizes the string input before being validated.</summary>
+    /// <param name="str">
+    /// The input string to validate.
+    /// </param>
+    /// <param name="formatProvider">
+    /// The optional format provider.
+    /// </param>
+    [Pure]
+    public virtual string NormalizeInput(string? str, IFormatProvider? formatProvider)
+        => str?.Trim() ?? string.Empty;
+
+    /// <summary>Returns a formatted <see cref="string" /> that represents the 
+    /// unknown state of the Single Value Object.
+    /// </summary>
+    /// <param name="format">
+    /// The format that this describes the formatting.
+    /// </param>
+    /// <param name="formatProvider">
+    /// The format provider.
+    /// </param>
+    [Pure]
+    public virtual string FormatUnknown(string? format, IFormatProvider? formatProvider) => "?";
+
+    /// <summary>Returns a formatted <see cref="string" /> that represents the Single Value Object.</summary>
+    /// <param name="str">
+    /// The string representing the Single Value Object.
+    /// </param>
+    /// <param name="format">
+    /// The format that this describes the formatting.
+    /// </param>
+    /// <param name="formatProvider">
+    /// The format provider.
+    /// </param>
+    /// <remarks>
+    /// Not called for empty and unknown.
+    /// </remarks>
+    [Pure]
+    public virtual string Format(string str, string? format, IFormatProvider? formatProvider) => str;
+
+    /// <summary>Serializes the Single Value Object to a JSON string.</summary>
+    /// <param name="str">
+    /// The string representing the Single Value Object.
+    /// </param>
+    [Pure]
+    public virtual string? ToJson(string? str)
+    {
+        if (str is null) return null;
+        else if (str == unknown) return FormatUnknown(default, CultureInfo.InvariantCulture);
+        else return str;
+    }
+
+    /// <summary>Serializes the Single Value Object to an XML string.</summary>
+    /// <param name="str">
+    /// The string representing the Single Value Object.
+    /// </param>
+    [Pure]
+    public virtual string? ToXml(string? str)
```

Comment:
Exactly the same as ToJson?

Reply:
Yes, but it is up to the user/implementer to keep those in sync (or not).